src/mgmt/srv_drpc.c

```diff
@@ -630,11 +628,10 @@ ds_mgmt_drpc_pool_evict(Drpc__Call *drpc_req, Drpc__Response *drpc_resp)
 
 	D_INFO("Received request to evict pool connections %s\n", req->id);
 
-	rc = uuid_parse(req->id, uuid);
-	if (rc != 0) {
-		D_ERROR("Unable to parse pool UUID %s: "DF_RC"\n", req->id,
-			DP_RC(rc));
-		D_GOTO(out, rc = -DER_INVAL);
+	if (uuid_parse(req->id, uuid) != 0) {
```

Comment:
given we repeat this with the same error message so frequently, why not create a pool_uuid_parse helper to return INVAL and print the invalid error?

Reply:
I did wonder that but getting it into a macro to keep the right line number and crucially function name would be tricky, it was a borderline decision though.  Some of the UUIDs are also not for pools